spec/features/tagging_history_spec.rb

```diff
@@ -0,0 +1,97 @@
+require "rails_helper"
```

Comment:
need some tests for the 'show' method of the controller

Reply:
I've added some tests in d1f6f9f

Reply:
Make that 1 test.